Lib/distutils/tests/test_config_cmd.py

```diff
@@ -47,7 +47,7 @@ def test_search_cpp(self):
         cmd = config(dist)
         cmd._check_compiler()
         compiler = cmd.compiler
-        is_xlc = shutil.which(compiler.preprocessor[0]).startswith("/usr/vac")
+        is_xlc = "xlc" in compiler.preprocessor[0].lower()
```

Comment:
What is compiler.preprocessor[0] value on AIX? Why do you need to convert to lower case? Can you use os.path.basename(path) == "xlc" rather than "xlc" in path?

Reply:
with `export CC=xlc_r`

On my server, with version 11 of the compiler compiler.preprocessor[0] is /usr/vac/bin/xlc_r. The alternative would be /usr/vacpp/bin/xlc_r

On a server I am guest, with versions 13 and 16 installed the compiler.preprocessor[0] value could be any of:   /opt/IBM/xlc/13.1.3/bin/xlc_r, /opt/IBM/xlC/13.1.3/bin/xlc_r, /opt/IBM/xlc/16.1.0/bin/xlc_r, or  /opt/IBM/xlC/16.1.0/bin/xlc_r 

However, if I chose just cc the potential values would be:
/opt/IBM/xlc/13.1.3/bin/cc, /opt/IBM/xlC/13.1.3/bin/cc, /opt/IBM/xlc/16.1.0/bin/cc, or  /opt/IBM/xlC/16.1.0/bin/cc

so, the choice for using lower() is to find compiler names that may be reached with xlC in the path.

Reply:
I'm not comfortable with checking for "xlc" substring anywhere in the path on any platform, but I would be fine if the test is made specific to AIX.